### ISSUE TITLE
Native Ads URLs return WebP Images which Unity can't handle

### ISSUE THREAD

**Issue by stefanwagnerdev** [uc-union/union-ads-sdk-unity-demo#2]
mNativeAd.getIconUrl() returns WebP image files which Unity can't load by default. 
Can I fetch only png or jpg images?

**Comment by leeowenowen** [comment]
@bompo  do you integration native ad with our unity ads sdk ?

**Comment by stefanwagnerdev** [comment]
Yes, I use the SDK from the github.com repository. The demo code doesn't work as well for native ads.

**Comment by leeowenowen** [comment]
ok, can you tell me the name of your BD, I can ask him/her to change configuration for you to remove the webp resource.

**Comment by stefanwagnerdev** [comment]
I tried to contact Amber Chan and Daisy Lin on the behalf ob Mobile Heroes but they didn't answered me so far (since 3 weeks) . So I tried to integrate the SDK beforehand. Maybe you can give me a direct contact?
